app/connector/servicenow/src/test/java/io/syndesis/connector/servicenow/ServiceNowMetaDataExtensionTest.java

```diff
@@ -24,18 +24,21 @@
 import org.apache.camel.CamelContext;
 import org.apache.camel.component.extension.MetaDataExtension;
 import org.apache.camel.component.servicenow.ServiceNowConstants;
+import org.apache.camel.component.servicenow.ServiceNowMetaDataExtensionWrapper;
 import org.apache.camel.impl.DefaultCamelContext;
 import org.assertj.core.api.Condition;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class ServiceNowMetaDataExtensionTest {
 
     @Test
+    @Disabled("This is not a unit test but an integration test")
     public void retrieveTableListTes() {
         CamelContext context = new DefaultCamelContext();
-        ServiceNowMetaDataExtension extension = new ServiceNowMetaDataExtension(context);
+        MetaDataExtension extension = ServiceNowMetaDataExtensionWrapper.getServiceNowMetaDataExtension(context);
```

Comment:
I think you can get the meta extension from the ServicewNow component now

Reply:
Ah, then that's why the constructors are protected?

Reply:
yes, it is not supposed to be used outside the component

Reply:
But the class is also protected:
https://github.com/apache/camel/blob/master/components/camel-servicenow/camel-servicenow-component/src/main/java/org/apache/camel/component/servicenow/ServiceNowMetaDataExtension.java#L59

That's why I am using MetadataExtension as the return value here: I cannot even import it.

Reply:
Something like that should work:

```java
var component = context.getComponent("salesforce", SalesforceComponent.class);
var meta = component.getExtension(MetaDataExtension.class);
```

Reply:
Changed according to this.